### PR TITLE
stf repo url fix

### PR DESCRIPTION
**Pull request by pinikomarov**
- **fix stf tests repo url**

### CODE REVIEW

**Review by lnatapov — CHANGES_REQUESTED** [review]
We should checkout branch old_stf_job
git clone -b old_stf_job https://github.com/infrawatch/feature-verification-tests.git

**Review by lnatapov — APPROVED** [review]
LGTM

File: ansible/osp_stf_tests.yaml [hunk 1]
```diff
@@ -14,7 +14,7 @@
   - name: Clone stf tests
     shell: |
       #!/bin/bash
-      oc exec -n openstack openstackclient -- bash -c "test -d /home/cloud-admin/functional-tests || (pushd /home/cloud-admin/ && git clone -c http.sslVerify=false https://github.com/infrawatch/functional-tests.git)"
+      oc exec -n openstack openstackclient -- bash -c "test -d /home/cloud-admin/feature-verification-tests || (pushd /home/cloud-admin/ && git clone -b old_stf_job -c http.sslVerify=false https://github.com/infrawatch/feature-verification-tests.git)"
```
Comment (olliewalsh) [inline ansible/osp_stf_tests.yaml:17]: `http.sslVerify=false` needs to be removed, if github.com fails SSL verification do not pull code from it!!!

**Review by olliewalsh — CHANGES_REQUESTED** [review]

**Comment by olliewalsh** [comment]
Re-running CI on this, failed with DNS issues

**Comment by pinikomarov** [comment]
@olliewalsh so this is good to go correct ? 
cc: @lnatapov

**Pull request merged**